packages/gatsby-telemetry/src/telemetry.js

```diff
@@ -312,12 +274,10 @@ module.exports = class AnalyticsTracker {
       skewness: !Number.isNaN(skewness) ? skewness : 0,
     }
   }
-
   async sendEvents() {
     if (!this.isTrackingEnabled()) {
-      return Promise.resolve()
+      return Promise.resolve(true)
     }
-
     return this.store.sendEvents()
   }
 }
```

Comment:
shouldn't this file be deleted?? why do we have a .js and a .ts?

Reply:
Done